.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: release
+on:
+  schedule:
+    - cron: '0 2 * * *' # UTC 2:00 every day
```

Comment:
@xgaozoyoe Do we plan to publish package now? I think our code is still in an early stage.

Reply:
It is only the cli tools for users to easily test.

Reply:
let's wait for another 2 months before publishing I guess. We can mark this PR WIP.

Reply:
I prefer to keep the workflow_dispatch and remove the schedule.

Reply:
> I prefer to keep the workflow_dispatch and remove the schedule.

LGTM, WDYT @xgaozoyoe

Reply:
shall we remove the schedule but manually do a release instead?